<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWindow</class>
 <widget class="QWidget" name="TeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Завуч</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>711</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Группы</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>701</width>
       <height>631</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>347</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Группа</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_groups">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_groups">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_upload_groups">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выгрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_groups">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Расписание</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>701</width>
       <height>541</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>87</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Учитель</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вт</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ср</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Чт</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пт</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вс</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_shedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_upload_shedule">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выгрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_shedule">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_shedule">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Преподаватели</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>701</width>
       <height>541</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>347</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_teachers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_upload_teachers">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выгрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_teachers">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_teachers">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Ученики</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>701</width>
       <height>541</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>230</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Группа</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_students">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_students">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_upload_students">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выгрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_students">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>652</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
